tools/terraform-bundle/README.md

```diff
@@ -119,13 +121,45 @@ this composite version number so that bundle archives can be easily
 distinguished from official release archives and from each other when multiple
 bundles contain the same core Terraform version.
 
-To include custom plugins in the bundle file, create a local directory "./plugins"
-and put all the plugins you want to include there. Optionally, you can use the
-`-plugin-dir` flag to specify a location where to find the plugins. To be recognized
-as a valid plugin, the file must have a name of the form
-`terraform-provider-<NAME>_v<VERSION>`. In
-addition, ensure that the plugin is built using the same operating system and
-architecture used for Terraform Enterprise. Typically this will be `linux` and `amd64`.
+
+## Custom Plugins
+To include custom plugins in the bundle file, create a local directory
+"./plugins" and put all the plugins you want to include there, under the
```

Comment:
I spotted a few instances of the `./plugins` vs `./.plugins` mixup which left me confused when doing local testing:

```suggestion
"./.plugins" and put all the plugins you want to include there, under the
```

Reply:
Ha! I'm starting to see why I lost so much time to that difference, too! When I moved it into a variable, I must've copied that from the docs 🙄